### ISSUE TITLE
Buzzer for RDZS

### ISSUE THREAD

**Issue by RegsJR** [dl9rdz/rdz_ttgo_sonde#67]
Hello DL9RDZ, please, would it be possible to add support for a piezzo passive buzzer on TBeam and wifi-only TTGO? Make the buzzer beep when is the module able to decode the sonde?

**Comment by dl9rdz** [comment]
Currently you can use the LED pin with an active buzzer. Using it with a passive buzzer would just require changing a few lines of code in flashLed() in RX_FSK.cpp, maybe with a configuration option. I'll keep this idea on the list of possible future enhancements.

**Comment by RegsJR** [comment]
Thank you very much for support! I will try. So, LED pin and the ground pin, am I right?

**Comment by dl9rdz** [comment]
Yes. The pin configured as LED pin is pulled HIGH for 700ms (within 1s) if a correct frame was decoded, and for 100ms if the start of a frame was detected, but not correctly decoded (CRC error for most types of sondes).

**Comment by gariac** [comment]
I don't see a LED pin on the ttbeam 1.1.  Is there an equivalent?

**Comment by 73-de-LZ** [comment]
It is by default on IO4, the red led near blue charging led, it falls under the OLED usually and it is connected to Vcc 3.3V instead to ground, so my advice is to add external LED, for example on port IO25 trough 2K to GND and configure it on "Hardware configuration"/"LED output port" = 25. 

As for the buzzer it is easy to add an active one as Hansi advice (active who generates fixed freq. sound when powered) with micro switch, or you can switch it off by LED output port" = -1, but this will switch off the LED also.
